cmd/otel-allocator/main.go

```diff
@@ -9,18 +9,21 @@ import (
 	"os/signal"
 	"syscall"
 
+	"github.com/open-telemetry/opentelemetry-operator/cmd/otel-allocator/allocation"
```

Comment:
This should be in the third group

Reply:
Gah! My goimports didn't work :( i'll do this manually